Makefile

```diff
@@ -0,0 +1,51 @@
+artifact_name       := chips-rest-interfaces-consumer
+version             := "unversioned"
+
+.PHONY: all
+all: build
+
+.PHONY: clean
+clean:
+	mvn clean
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+	rm -f ./build.log
+
+.PHONY: build
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	mvn package -DskipTests=true
+	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
+
+.PHONY: test
+test: test-unit
+
+.PHONY: test-unit
+test-unit: clean
+	mvn test -Dgroups="Unit"
```

Comment:
I'd remove the groups argument for now as there is not currently any groups in the project

Reply:
Removed argument